notebooks/demo.ipynb

```diff
@@ -7,6 +7,16 @@
     "# HOW TO ADD A NEW CLASS TO OBJECT DETECTION PIPELINE?"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## Uncomment command below to kill current job:\n",
+    "#!neuro kill $(hostname)"
```

Comment:
AFAIU, the hostname will be something like `obj-det--mariyadydova` (i.e. contain the job name and the username). How would that be enough to kill a job?

Reply:
no. The hostname is always job-ID

Reply:
job's URL is `obj-det--mariyadydova.jobs.neu.ro`

Reply:
OK.